bark/models/behavior/idm/base_idm.hpp

```diff
@@ -97,14 +99,14 @@ class BaseIDM : virtual public BehaviorModel {
   const float GetDesiredTimeHeadway() const {
     return param_desired_time_head_way_;
   }  // unit is seconds
-  const float GetMaxAcceleration() const {
+  const float GetLonAccelerationMax() const {
     return param_max_acceleration_;
   }  // unit is meter/second^2
   const float GetAccelerationLowerBound() const {
-    return param_acceleration_lower_bound_;
+    return acceleration_limits_.lon_acc_min;
   }
   const float GetAccelerationUpperBound() const {
-    return param_acceleration_upper_bound_;
+    return acceleration_limits_.lon_acc_max;
```

Comment:
and here in `GetAccelerationLonLowerBound()` and `GetAccelerationLonUpperBound()`